java-client/src/test/java/org/opensearch/client/opensearch/core/search/InnerHitsResultTest.java

```diff
@@ -0,0 +1,84 @@
+package org.opensearch.client.opensearch.core.search;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.io.StringReader;
+import org.junit.Test;
+import org.opensearch.client.json.JsonData;
+import org.opensearch.client.json.JsonpMapper;
+import org.opensearch.client.json.jsonb.JsonbJsonpMapper;
+
+public class InnerHitsResultTest {
```

Comment:
Do we have unit tests for outer hits with nullable ? Should be allowed or not allowed ?